StandardisedGrasp grasp_message
string grasp_name
---
bool success
